src/main/java/com/auth0/client/mgmt/ManagementAPI.java

```diff
@@ -27,8 +30,9 @@
      *
      * @param domain   the tenant's domain.
      * @param apiToken the token to authenticate the calls with.
+     * @param options  configuration options for this client instance.
      */
-    public ManagementAPI(String domain, String apiToken) {
+    public ManagementAPI(String domain, String apiToken, ClientOptions options) {
```

Comment:
Similar to the above comment, should update the method JavaDoc here and add a `@see` for the constructor that does not take a `ClientOptions`.